### PR TITLE
Check info status before displaying info icon in search bar

### LINKED ISSUE #20409: Search page - Double error/warning icons
## Description
When there is a warning/error for a search, 2 icons appear in the UI:
image
image

## Steps To Reproduce

1. Navigate to search page
2. Enter a bad search query

## Environment

- Graylog Version: 6.1.0-beta.1
- OpenSearch Version:
- MongoDB Version:
- Browser Version: Firefox & Chrome LTS

### PR DESCRIPTION
**Pull request by ousmaneo**
Check query validation status info for the Info error icon.

resolve #20409

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: changelog/unreleased/issue-20409.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type = "fixed"
+message = "Fix search bar query validation error showing double icons."
+
+issues = ["20409"]
+pulls = [""]
```
Comment (gally47) [inline changelog/unreleased/issue-20409.toml:5]: Shouldn't we add this PR # here?

**Review by gally47 — APPROVED** [review]
LGTM, thanks for fixing this.

File: graylog2-web-interface/src/views/components/searchbar/queryvalidation/QueryValidation.tsx [hunk 2]
```diff
@@ -212,7 +212,7 @@ const QueryValidation = () => {
                                 $clickable
                                 tabIndex={0}
                                 type="button">
-              <ErrorIcon $status={status} name={isInfo ? 'database' : 'error'} />
+              {isInfo && <ErrorIcon $status={status} name={isInfo ? 'database' : 'error'} />}
```
Comment (gally47) [inline graylog2-web-interface/src/views/components/searchbar/queryvalidation/QueryValidation.tsx:215]: `isInfo` is always true here, so we can just set it to 'database' right?

Comment (ousmaneo) [reply graylog2-web-interface/src/views/components/searchbar/queryvalidation/QueryValidation.tsx]: Good findings, this was the result of using different icon components during development iteration.

**Review by gally47 — APPROVED** [review]

**Pull request merged**